packages/terra-clinical-result/src/common/other/_KnownNoData.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import styles from './OtherTemplates.module.scss';
+
+const cx = classNames.bind(styles);
+
+const KnownNoData = (props) => {
+  const {
+    ...customProps
+  } = props;
+
+  const templateClassnames = cx([
+    'no-data',
+  ]);
+
+  return (
+    <React.Fragment>
```

Comment:
You should be able to remove the `<React.Fragment>` wrapper of the span

Reply:
Removed https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-ee206f19364f5e67dbef6f2d248bc1a6R17